test_settings.py

```diff
@@ -1,6 +1,22 @@
+# -*- coding: utf-8 -*-
+import os
+
+
 DEBUG = True
-SECRET_KEY = 'x'
```

Comment:
I just had them already while you added another test settings into the repo. So pretty much just decided to use my existing version.